### PR TITLE
Loosen dependency on activesupport to allow Rails 5

### PR DESCRIPTION
**Pull request by agrobbin**
- [x] #34

### CODE REVIEW

**Comment by MDaubs** [comment]
I suggest we also run tests against Rails 5.

**Comment by agrobbin** [comment]
@MDaubs 👍  done!

**Comment by agrobbin** [comment]
Ergh, looks like we are only testing against Ruby 2.1.2, which Rails 5+ doesn't support. I will issue a separate PR to add a Ruby version matrix.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+  - 2.3.0
+gemfile:
+  - gemfiles/rails_4.1.gemfile
+  - gemfiles/rails_4.2.gemfile
+  - gemfiles/rails_5.0.gemfile
+script: bundle exec rspec spec
+matrix:
+  fast_finish: true
+  exclude:
+    - rvm: 2.1.10
```
Comment (MDaubs) [inline .travis.yml]: Needs to be `2.1.8`?

**Pull request merged**